src/libs/dps-cache-starter/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/cache/RedisStorageService.java

```diff
@@ -62,5 +62,19 @@ public String put(byte[] content) {
 
     }
 
+    /**
+     * Removes a document from redis key value store.
+     * @param key    object key to evict from storage
+     */
+    @Override
+    public void delete(String key) {
+
+        try {
+            this.cacheManager.getCache(Keys.DPS_CACHE_NAME).evict(key);
```

Comment:
Can you add unit tests in RedisStorageServiceTests: https://github.com/bcgov/jag-dps/blob/76bfa478dbcbf9c6ff808477510ad83345c16581/src/libs/dps-cache-starter/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/cache/RedisStorageServiceTest.java#L15